src/main/java/com/renatusnetwork/momentum/commands/SquadsCMD.java

```diff
@@ -0,0 +1,272 @@
+			case "warp":
+				Level level = player.getLevel();
+
+				if (squad == null)
+					noSquad(sender);
+				else if (!SquadsManager.isLeader(player))
+					player.sendMessage(Utils.translate("&cYou are not the squad leader!"));
+				else if (squad.hasWarpCooldown())
+					player.sendMessage(Utils.translate("&cWarp is on cooldown"));
+				else if (player.getPlayer().getWorld().getName().equalsIgnoreCase(Momentum.getSettingsManager().player_submitted_world))
+					player.sendMessage(Utils.translate("&cYou cannot warp to a plot!"));
+				else if (!player.inLevel())
+					player.sendMessage(Utils.translate("&cYou are not in a level!"));
+				else if (level.isAscendance())
+					player.sendMessage(Utils.translate("&cYou cannot warp to ascendance!"));
+				else if (level.isEventLevel() || level.isRaceLevel())
+					player.sendMessage("&cYou cannot warp to a race or event!");
+				else {
+					// notify first so failure messages send to individual players after
+					SquadsManager.notifyMembers(squad, "&9SC &3" + player.getDisplayName() + " &bhas warped to " + level.getFormattedTitle());
+					squadsManager.warpMembers(player);
+
+					squadsManager.notifySqChatSpies(String.format("&3%s &9has warped their party to &3%s", player.getDisplayName(), level.getFormattedTitle()));
+				}
+
+				break;
+			default:
+				sendHelp(sender);
+		}
+
+		return true;
+	}
+
+	private void sendHelp(CommandSender sender) {
+		sender.sendMessage(Utils.translate("&9-- Help --"));
+		sender.sendMessage(Utils.translate("&9Command Aliases: &b[squad, sq]"));
+		sender.sendMessage(Utils.translate("&3/squads [help]  &bdisplays this menu"));
+		sender.sendMessage(Utils.translate("&3/squads list  &blists all players in squad"));
+		sender.sendMessage(Utils.translate("&3/squads invite <player>  &binvites player to squad (creates squad if not in one)"));
+		sender.sendMessage(Utils.translate("&3/squads accept <player>  &baccepts squad invite from player"));
+		sender.sendMessage(Utils.translate("&3/squads leave  &bleaves current squad"));
+		sender.sendMessage(Utils.translate("&3/squads kick <player>  &bkicks player from the squad"));
+		sender.sendMessage(Utils.translate("&3/squads disband  &bdisbands squad"));
+		sender.sendMessage(Utils.translate("&3/squads promote <player>  &btransfers ownership of squad to player"));
+		sender.sendMessage(Utils.translate("&3/squads chat  &btoggles squad chat"));
+		sender.sendMessage(Utils.translate("   &9Aliases: &b[sqc]"));
+		sender.sendMessage(Utils.translate("&3/squads chat <message> &bsends message to squad chat"));
+		sender.sendMessage(Utils.translate("   &9Aliases: &b[sqc <message>]"));
+		sender.sendMessage(Utils.translate("&3/squads warp  &bsends all players in squad to leader's level"));
+		if (sender.hasPermission("momentum.admin"))
+			sender.sendMessage(Utils.translate("&3/squads chatspy  &btoggles squad chat spy"));
+		sender.sendMessage(Utils.translate("&9---------"));
+	}
+
+	private void noSquad(CommandSender sender) {
```

Comment:
Can we maybe store this error message in a final at the top of the file and then just sender.sendMessage(final var)? I'm against this because it adds unneeded functionality since it's a function call that does one line task if that makes sense